src/Test/Perf/util/RelativeDirectory.cs

```diff
@@ -23,27 +23,13 @@ public RelativeDirectory([CallerFilePath] string workingFile = "")
 
         public string MyWorkingDirectory => _workingDir;
 
-
-        /// Returns the directory that you can put artifacts like
-        /// etl traces or compiled binaries
-        public string MyArtifactsDirectory
-        {
-            get
-            {
-                var path = Path.Combine(MyWorkingDirectory, "artifacts");
-                Directory.CreateDirectory(path);
-                return path;
-            }
-        }
-
-        public string MyTempDirectory
+        public string TempDirectory
         {
             get
             {
-                var workingDir = MyWorkingDirectory;
-                var path = Path.Combine(workingDir, "temp");
-                Directory.CreateDirectory(path);
-                return path;
+                var tempDirectory = Path.Combine(Environment.ExpandEnvironmentVariables("%SYSTEMDRIVE%")+@"\", "PerfTemp");
```

Comment:
`%USERPROFILE%`has the same issue of rooting our test solution deep inside and it will result in Path names longer than 265 characters limit error again. Thats why I wanted the minimal root path. Also our previous test infra had test solution in C:\Roslyn\